client/src/partials/accounts/edit/accounts.edit.js

```diff
@@ -136,6 +136,19 @@ function AccountEditController($rootScope, $state, AccountStore, Accounts, Notif
   }
 
 
+  vm.titleChangedValidation = titleChangedValidation;
+
+  // @todo form validation using validators on a component
+  function titleChangedValidation(newAccountType) {
+    console.log(newAccountType, Constants.accounts.TITLE);
+    if ((Number(newAccountType) !== Constants.accounts.TITLE) && vm.account.children.length) {
```

Comment:
It would be easier to read if these were variables:
 ```js
var isTitleAccount = Number(newAccountType) === Constants.accounts.TITLE;
var hasChildren = vm.account.children.length;

if (!isTitleAccount && hasChildren) {
  // etc
}
```